src/editor/styles/globals.scss

```diff
@@ -1,7 +1,11 @@
 @tailwind base;
 @tailwind components;
 @tailwind utilities;
-@import "settings";
+@import url(https://sindresorhus.com/github-markdown-css/github-markdown.css);
```

Comment:
If this person ever decides to move the file we're screwed. Best is to download the file and add it as a separat style sheet. You can then import this directly in the component where you need it. Otherwise it is loaded on every page.